modules/node/handler/nodes_handler.e

```diff
@@ -49,10 +52,68 @@ feature -- HTTP Methods
 			create {GENERIC_VIEW_CMS_RESPONSE} l_page.make (req, res, api)
 			l_page.add_variable (node_api.nodes, "nodes")
 
+			create pager.make (api, node_api)
+			number_of_pages := (node_api.nodes_count // pager.limit) + 1
+
+				-- Size:limit
+			if
+				attached {WSF_STRING} req.query_parameter ("size") as l_size and then
+				l_size.is_integer
+			then
+				pager.set_limit (l_size.integer_value.to_natural_32)
+			end
+
+
+
+				--Page:offset
+			if
+				attached {WSF_STRING} req.query_parameter ("page") as ll_page and then
```

Comment:
Ok then you could rename
l_page: CMS_RESPONSE
to

l_response: CMS_RESPONSE

and use l_page for the object test local variable name